stubs/Helpers.stub

```diff
@@ -155,3 +155,17 @@ function __($key = null, $replace = [], $locale = null) {}
  * @return ($callback is null ? mixed : ($value is null ? null : TReturn))
  */
 function optional($value = null, callable $callback = null) {}
+
+/**
+ * Transform the given value if it is present.
+ *
+ * @template TValue of mixed
+ * @template TReturn of mixed
+ * @template TDefault of mixed
+ *
+ * @param TValue $value
+ * @param callable(TValue): TReturn $callback
+ * @param null|TDefault|callable(TValue): TDefault $default
+ * @return ($value is empty ? ($default is null ? null : TDefault) : TReturn)
```

Comment:
Isn't `$default is null ? null : TDefault` the same as just `TDefault`?

Reply:
Also, transform has another logic, different from empty: it uses `blank()` under the hood. Unlike `empty`, these values a not blank:
 - `false`
 - `"0"`
 - `0`
 - `0.0`

Some values are blank, but not empty:
 - `Countable` with `count() === 0` are empty also (e.g. empty Collection)
 - strings contains whitespaces only